internal/cli/opsmanager/logs/list_key_provider.go

```diff
@@ -30,7 +30,7 @@ type KeyProviderListOpts struct {
 	fs   afero.Fs
 }
 
-var listTmpl = `{{ range . }}{{ .Provider }}:{{if .Filename}} Filename = {{ .Filename }}{{end}}{{if .UniqueKeyID}} Unique Key ID = "{{ .UniqueKeyID }}"{{end}}{{if .KMIPServerName}} KMIP Server Name = "{{ .KMIPServerName }}"{{end}}{{if .KMIPPort}} KMIP Port = "{{ .KMIPPort }}"{{end}}{{if .KeyWrapMethod}} Key Wrap Method = "{{ .KeyWrapMethod }}"{{end}}
+var listTmpl = `{{ range . }}{{ .Provider }}:{{if .Filename}} Filename = {{ .Filename }}{{end}}{{if .UID}} Unique Key ID = "{{ .UID }}"{{end}}{{if .KMIPServerName}} KMIP Server Name = "{{index .KMIPServerName 0}}"{{end}}{{if .KMIPPort}} KMIP Port = "{{ .KMIPPort }}"{{end}}{{if .KeyWrapMethod}} Key Wrap Method = "{{ .KeyWrapMethod }}"{{end}}
```

Comment:
checking why the header format changed with server team

Reply:
for now, there won't be any other header change. Adapted the code to support multiple server names:
https://jira.mongodb.org/browse/CLOUDP-116151?focusedCommentId=4431895&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4431895